pkg/cmd/space/create/create.go

```diff
@@ -48,7 +48,7 @@ type CreateOptions struct {
 }
 
 func NewCreateOptions(f factory.Factory, flags *CreateFlags, c *cobra.Command) *CreateOptions {
-	dependencies := cmd.NewDependencies(f, c)
+	dependencies := cmd.NewSystemDependencies(f, c)
```

Comment:
Was that all it was!! Noice!

Reply:
Yep 😆, I'll leave this to be bundled in the next release someone needs to push out.